usefulUtilities/spawnPointZone/README.md

```diff
@@ -0,0 +1,22 @@
+# SPAWN POINT ZONE
+
+## Description
+This utility handles users falling through the floor when first loading in to HIFI. When the user loads this zone, they will be moved to random position within a cube specified by its position and dimensions in the userData of this zone.
+
+## File List
+### Client Side
+* **spawnPointZone.js** attached to the zone around the domain. The userData of this zone should specify the position and dimensions of an area in which users should spawn at.
```

Comment:
please specify an example userdata. see [here](https://github.com/highfidelity/hifi-content/blob/master/usefulUtilities/avatarEntityRemover/README.md) for an example of the format i'd like to see in this readme.